platform/pulp/platform/fields.py

```diff
@@ -0,0 +1,96 @@
+"""
+Custom django fields provided by pulp platform
+"""
+import json
+
+from django.core.exceptions import ValidationError
+from django.db import models
+from django.utils.translation import six, ugettext_lazy as _
+
+
+def _parse_json(json_string):
+    """
+    Parse a json string
+
+    :param json_string: A json string to parse
+    :type json_string: str
+
+    :return: A python object built from the json string
+    :rtype: Any
+    """
+    try:
+        return json.loads(json_string)
+    except Exception:
+        raise ValidationError(_("Invalid JSON stored in database"))
+
+
+class JSONField(models.TextField):
+    """
+    A custom django field to serialize dicts into a text field
+
+    """
+    def from_db_value(self, value, expression, connection, context):
+        """
+        Converts a value as returned by the database to a Python object
+
+        :param value: The value to convert to Python
+        :type value: Any
+
+        :param expression:
+        :param connection:
+        :param context:
```

Comment:
Consider these two resolutions to this docstring issue. One would be to adjust the function signature to be:

`from_db_value(self, value, *args, **kwargs):`

And then document it with:

```
:param args: unused positional arguments
:type args: list
:param kwargs: unused keyword arguments
:type kwargs: dict
```
This has the advantage of being resilient to most signature changes to the function as Django evolves. I've had this problem before with Django specifically. I think I prefer this option.

The other option is to link to the docs. If you do that I recommend pointing at latest and not just the 1.8 version since one day we will upgrade.